src/test/java/uk/gov/justice/hmpps/prison/util/JwtAuthenticationHelper.kt

```diff
@@ -0,0 +1,49 @@
+package uk.gov.justice.hmpps.prison.util
+
+import io.jsonwebtoken.Jwts
+import io.jsonwebtoken.SignatureAlgorithm
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Primary
+import org.springframework.security.oauth2.jwt.JwtDecoder
+import org.springframework.security.oauth2.jwt.NimbusJwtDecoder
+import java.security.KeyPair
+import java.security.KeyPairGenerator
+import java.security.interfaces.RSAPublicKey
+import java.util.Date
+import java.util.UUID
+
+@Configuration
+class JwtAuthenticationHelper {
+  private val keyPair: KeyPair
+
+  init {
+    val gen = KeyPairGenerator.getInstance("RSA")
```

Comment:
switch to generating the key pair here (like all other services) for tests rather than loading from application properties